pkg/config/analysis/analyzers/externalcontrolplane/externalcontrolplane.go

```diff
@@ -53,6 +54,25 @@ const (
 
 // Analyze implements Analyzer
 func (s *ExternalControlPlaneAnalyzer) Analyze(c analysis.Context) {
+	reportWebhookURL := func(r *resource.Instance, hName string, clientConf v1.WebhookClientConfig) bool {
+		// If defined, it means that an external istiod has been adopted
+		if clientConf.URL != nil {
+			result, err := lintWebhookURL(*clientConf.URL)
+			if err != nil {
+				c.Report(gvk.ValidatingWebhookConfiguration, msg.NewInvalidExternalControlPlaneConfig(r, *clientConf.URL, hName, err.Error()))
+				return false
```

Comment:
Why we are returning false? Return false means no iteration anymore, and I think we should always return true here.

Reply:
yes, need still return true, to continue for each